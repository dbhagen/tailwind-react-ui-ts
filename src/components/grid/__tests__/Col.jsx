import React from 'react'
import { shallow } from 'enzyme'

import { Col } from '..'

const setup = (testProps = {}) => {
  const props = Object.assign({}, testProps)

  const wrapper = shallow(<Col {...props}>Content</Col>)

  return {
    props,
    wrapper,
  }
}

describe('Col', () => {
  it('renders matching snapshot', () => {
    const { wrapper } = setup({ size: { def: 'full', sm: '1/2', md: '1/4' } })

    expect(wrapper).toMatchSnapshot()
  })
})
