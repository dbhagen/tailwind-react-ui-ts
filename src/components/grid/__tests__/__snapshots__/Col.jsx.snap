// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Col renders matching snapshot 1`] = `
<Box
  inline={false}
  inlineBlock={false}
  is="li"
  size={
    Object {
      "def": "full",
      "md": "1/4",
      "sm": "1/2",
    }
  }
  w="full"
>
  Content
</Box>
`;
